Examples/calculator.lisp

```diff
@@ -138,3 +149,6 @@
              (list seven eight divide)
              (list nine zero result))))))
   (:top-level (calculator-frame-top-level . nil)))
+
+#+nil
+(calculator)
```

Comment:
Oh, sorry.

I only meant for the first commit to be in the PR.... the other was just to show what I had to do to get a button colored.

Sorry about the confusion!